cmd/queue/main.go

```diff
@@ -214,10 +225,12 @@ func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.H
 			return
 		case network.IsKubeletProbe(r):
 			// Do not count health checks for concurrency metrics
-			handler.ServeHTTP(w, r)
+			handler.ServeHTTP(recorder, r.WithContext(reqCtx))
 			return
 		}
-
+		probeSpan.End()
```

Comment:
Do we need/want to add failure information to the probeSpan in the error cases above? Specifically in the "container not ready" case. As it stands the span will always be a "success" (though the outer span will be a failure).

Reply:
I added error annotations here, which will cause the spans to show up as red in zipkin, rather than successful/green. I added error annotations for each of the failure cases as well as tests to check for these.